src/VAP/BGSObjectDetector/BGSObjectDetector.csproj

```diff
@@ -1,11 +1,11 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFramework>netcoreapp2.2</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="OpenCvSharp4" Version="4.0.0.20190108" />
+    <PackageReference Include="OpenCvSharp4" Version="4.1.1.20191110" />
```

Comment:
📝 I'm planning to refactor all the version references to a **Versions.props** file so they are centrally-managed for this repository, but I did not make this change as part of this pull request.

Reply:
Good point. Can you please add the change to this PR?

Reply:
Done!